lib/notesclub_web/live/notesLive.ex

```diff
@@ -0,0 +1,200 @@
+defmodule NotesclubWeb.NotesLive do
+  use NotesclubWeb, :live_view
+  use Phoenix.HTML
+  import Phoenix.LiveView.Helpers
+  import Phoenix.LiveView
+
+  alias NotesclubWeb.NotesLive
+  alias Notesclub.Notebooks
+
+  @notebooks_in_home_count 7
+  def notebooks_in_home_count, do: @notebooks_in_home_count
+
+  def mount(_params, _session, socket) do
+    {:ok, assign(socket, notebooks: [], notebooks_count: 0, search: nil, page: :random)}
```

Comment:
```suggestion
    {:ok, socket}
```
Do we need to assign these variables here?
We're already setting them in `handle_params`.

Reply:
You are right, we dont need those assignments. Removed